web-client/src/app/views/delete-user/delete-user.page.ts

```diff
@@ -40,18 +44,47 @@ export class DeleteUserPage implements OnInit {
   /** @see validatedAddress */
   address?: string;
 
+  hideXrpBalance = environment.hideXrpBalance;
+
+  tokenIssuer = environment.tokenIssuer;
+
+  tokenSign: string | undefined;
+
+  /** A balance held by the current user. */
+  balances = this.sessionQuery.allBalances;
+
+  paymentOptions?: PaymentOption[];
+
+  checkAmountIsZero?: boolean;
+
   constructor(
     private navCtrl: NavController,
     private modalCtrl: ModalController,
     public sessionService: SessionService,
     public sessionQuery: SessionQuery,
     public notification: SwalHelper,
     public sessionAlgorandService: SessionAlgorandService,
-    public sessionXrplService: SessionXrplService
+    public sessionXrplService: SessionXrplService,
+    public deleteUserSrvice: DeleteUserService,
+    public purePayPageComponent: PurePayPageComponent
   ) {
     this.addressForm = new FormGroup({
       address: new FormControl('', [Validators.required, addressValidator]),
     });
+
+    if (this.hideXrpBalance) {
+      this.tokenSign = environment.tokenSymbol;
+    } else {
+      this.tokenSign = 'XRP';
+    }
+
+    this.sessionQuery.allBalances.subscribe((balances) => {
+      this.checkAmountIsZero = balances.some(
+        (amount) =>
+          amount.assetDisplay.assetSymbol === 'FOO' && amount.amount !== 0
```

Comment:
hardcoded `FOO`, do you have to change this?

Reply:
Thanks for catching this; it should be `environment.tokenSymbol`.